specification/oracle/resource-manager/Oracle.Database/preview/2023-09-01-preview/openapi.json

```diff
@@ -8447,16 +8866,42 @@
           {
             "name": "Restricted",
             "value": "Restricted",
-            "description": "RESTRICTED value"
+            "description": "Restricted permission level"
           },
           {
             "name": "Unrestricted",
             "value": "Unrestricted",
-            "description": "UNRESTRICTED value"
+            "description": "Unrestricted permission level"
           }
         ]
       }
     },
+    "PlanUpdate": {
```

Comment:
is this rename by design? the previous name is `ResourcePlanTypeUpdate`.

Reply:
Yes, that is by design to be consistent with `Azure.ResourceManager.Foundations.Plan` model name

https://github.com/Azure/azure-rest-api-specs/pull/29305/files#diff-34a4da4c732f7644203cef46fe88191428c928c907176d5e99fc5d6981a8479dR52-R54